src/components/gitOps/GitOpsConfiguration.tsx

```diff
@@ -212,40 +246,23 @@ export default class GitOpsConfiguration extends Component<GitOpsProps, GitOpsSt
             <span><a rel="noreferrer noopener" target="_blank" className="learn-more__href" href={DOCUMENTATION.GLOBAL_CONFIG_GITOPS}> Learn more about GitOps </a> </span></p>
             <form className="bcn-0 bw-1 en-2 br-8 pb-22 pl-20 pr-20">
                 <div className="login__sso-flex">
-                    <label className="tertiary-tab__radio">
-                        <input type="radio" name="status" value={GitProvider.Github} checked={this.state.tab === GitProvider.Github} onChange={this.handleGitopsTab} />
-                        <span className="tertiary-tab sso-icons">
-                            <aside className="login__icon-alignment"><GitHub /></aside>
-                            <aside className="login__text-alignment"> GitHub</aside>
-                            <div>
-                                {(this.state.lastActiveGitOp?.provider === GitProvider.Github) ? <aside className="login__check-icon"><img src={Check} /></aside> : ""}
-                            </div>
-                        </span>
-                    </label>
-                    <label className="tertiary-tab__radio">
-                        <input type="radio" name="status" value={GitProvider.GitLab} checked={this.state.tab === GitProvider.GitLab} onChange={this.handleGitopsTab} />
-                        <span className="tertiary-tab sso-icons">
-                            <aside className="login__icon-alignment"><GitLab /></aside>
-                            <aside className="login__text-alignment"> GitLab</aside>
-                            <div>
-                                {this.state.lastActiveGitOp?.provider === GitProvider.GitLab ? <aside className="login__check-icon"><img src={Check} /></aside> : ""}
-                            </div>
-                        </span>
-                    </label>
+                    <GitProviderTab tab={this.state.tab} handleGitopsTab={this.handleGitopsTab} lastActiveGitOp={this.state.lastActiveGitOp} provider={GitProvider.Github} gitops="GitHub" />
```

Comment:
we are passing provider and gitOps.. are both required? cant it be done with one?

Reply:
We are using`gitops` to write gitops name below icon : <img width="124" alt="Screenshot 2021-05-20 at 1 25 23 PM" src="https://user-images.githubusercontent.com/61836271/118940928-d79f1400-b96e-11eb-821c-45455b72a5f9.png">
for `Azure`  : 
in case of provider we need  `Azure` 
in case of gitops we need  `AZURE_DEVOPS`